docs/reference/upgrade/cluster_restart.asciidoc

```diff
@@ -1,15 +1,8 @@
 [[restart-upgrade]]
 == Full cluster restart upgrade
 
-A full cluster restart upgrade requires that you shut all nodes in the cluster
-down, upgrade them, and restart the cluster. A full cluster restart was required
-when upgrading to major versions prior to 6.x. Elasticsearch 6.x supports
-<<rolling-upgrades, rolling upgrades>> from *Elasticsearch 5.6*. Upgrading to
-6.x from earlier versions requires a full cluster restart. See the
-<<upgrade-paths,Upgrade paths table>> to verify the type of upgrade you need
-to perform.
-
-To perform a full cluster restart upgrade:
+To upgrade directly to {es} {version} from 6.6 or earlier, you must shut down all
+nodes in the cluster, upgrade each node to {version}, and restart the cluster:
```

Comment:
This `6.6 or earlier` is a bit confusing to me, since that implies you can use this procedure to upgrade directly from 5.x too.  I believe that's not true (i.e. you have to upgrade to 6.x and reindex your indices (including the internal ones) before that's possible.